apps/website/src/app/rating-shields/page.tsx

```diff
@@ -103,11 +129,93 @@ export default function RatingShieldsPage() {
               }}
             />
           </div>
-          <div></div>
+          <div className="flex items-center justify-center">
+            <NameBadge placeholder="notrab.eth" forceError />
+          </div>
           <div className="flex items-center justify-center">
             <NameBadge placeholder="notrab.eth" />
           </div>
         </div>
+        <div className="grid grid-cols-6 gap-x-6 py-5">
+          <div className="flex items-center font-mono">
+            <pre>{"<NameBadge onClick />"}</pre>
+          </div>
+
+          <div className="flex items-center justify-center">
+            <NameBadge
+              data={{
+                rating: "pass",
+                risk_count: 0,
+                highest_risk: null,
+                name: "lightwalker.eth",
+                namehash:
+                  "0x5c1f4e4189d173a562af8d27771e2a1394ccbfa466f0e72b429dd317afce4c06",
+                normalization: "normalized",
+                title: "Looks Good",
+                subtitle: "All security checks passed!",
+                beautiful_name: "lightwalker.eth",
+              }}
+              onClick={() => alert("Clicked")}
```

Comment:
So the issue here is that it will invoke `onClick` twice if you click the text because it's inside of another button thanks to the tooltip. Happy to try figure out another solution for this...

Reply:
Added `onClick` to the parent as well as the child.

I would prefer to refactor this whole component at some point so we can pass children as components that adjust the behaviours we want instead of prop drilling everything.